src/fu-keyring-utils.h

```diff
@@ -7,12 +7,12 @@
 #pragma once
 
 #include <xmlb.h>
+#include <jcat.h>
 
-#include "fu-keyring.h"
 #include "fwupd-enums.h"
 
-FuKeyring	*fu_keyring_create_for_kind		(FwupdKeyringKind kind,
-							 GError		**error);
 gboolean	 fu_keyring_get_release_flags		(XbNode		*release,
 							 FwupdReleaseFlags *flags,
 							 GError		**error);
+FwupdKeyringKind fu_keyring_kind_from_blob_kind		(JcatBlobKind	 kind);
```

Comment:
I would also question if there still needs to be a concept of a `FwupdKeyring/fu_keyring` anymore.  Instead maybe this should rename to something about getting jcat_release flags?  And returning jcat types?

Reply:
If we can nuke it from the remote files, I agree. I'll have a play.

Reply:
Played; removed.